src/libraries/System.Private.CoreLib/src/System/IO/TextReader.cs

```diff
@@ -203,18 +203,56 @@ public virtual int ReadBlock(Span<char> buffer)
         }
 
         #region Task based Async APIs
-        public virtual Task<string?> ReadLineAsync() =>
+        public virtual Task<string?> ReadLineAsync() => ReadLineCoreAsync(default);
+
+        /// <summary>
+        /// Reads a line of characters asynchronously and returns the data as a string.
+        /// </summary>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
+        /// <returns>A value task that represents the asynchronous read operation. The value of the <c>TResult</c>
+        /// parameter contains the next line from the text reader, or is <c>null</c> if all of the characters have been read.</returns>
+        /// <exception cref="ArgumentOutOfRangeException">The number of characters in the next line is larger than <see cref="int.MaxValue"/>.</exception>
+        /// <exception cref="ObjectDisposedException">The text reader has been disposed.</exception>
+        /// <exception cref="InvalidOperationException">The reader is currently in use by a previous read operation.</exception>
+        /// <remarks>
+        /// <para>The <see cref="TextReader"/> class is an abstract class. Therefore, you do not instantiate it in
+        /// your code. For an example of using the <see cref="ReadLineAsync(CancellationToken)"/> method, see the
+        /// <see cref="StreamReader.ReadLineAsync(CancellationToken)"/> method.</para>
+        /// <para>If the current <see cref="TextReader"/> represents the standard input stream returned by
+        /// the <c>Console.In</c> property, the <see cref="ReadLineAsync(CancellationToken)"/> method
```

Comment:
I had this originally, but kept getting `C:\dotnet\runtime\src\libraries\System.Private.CoreLib\src\System\IO\TextReader.cs(222,28): error CS1574: XML comment has cref attribute 'In' that could not be resolved [C:\dotnet\runtime\src\coreclr\System.Private.CoreLib\System.Private.CoreLib.csproj]`

Looks like the build is failing for the same reason, too. I also tried `<see cref="System.Console.In" />` but still no luck.

(Does the `System.Console` assembly depend on this one, because it references the `TextReader` type? Maybe this reference isn't possible?)

Reply:
I pushed a fix to try to make sure the build wasn't broken, but I don't know the appropriate long-term fix to get an actual reference in the docs to `Console.In`: https://github.com/dotnet/runtime/pull/61898/commits/6a36fd437b12147eb079a08945f79cc56658e71c